repository-data/local/src/main/resources/hcm-content/content/taxonomies/publication_taxonomy.yaml

```diff
@@ -332,6 +332,14 @@
         jcr:primaryType: hippotaxonomy:categoryinfos
       hippotaxonomy:key: outcomes-and-quality-indicators
       jcr:primaryType: hippotaxonomy:category
+    /patient-communication:
+      /hippotaxonomy:categoryinfos:
+        /en:
+          hippotaxonomy:name: Patient communication
+          jcr:primaryType: hippotaxonomy:categoryinfo
+        jcr:primaryType: hippotaxonomy:categoryinfos
+      hippotaxonomy:key: patient-communication
+      jcr:primaryType: hippotaxonomy:category
```

Comment:
Same with this file, you will need to make these changes in the console too.

Reply:
same comment as above, it is used only for the integration test to run against local data.